DIRECTORY.md

```diff
@@ -0,0 +1,22 @@
+<!--- DO NOT EDIT: This file is automatically generated by `directory.nim` -->
+
+# The Algorithms — Nim: Directory Hierarchy
+
+## Dynamic Programming
+  * [Catalan Numbers](dynamic_programming/catalan_numbers.nim)
+  * [Viterbi](dynamic_programming/viterbi.nim)
+
+## Maths
+  * [Area](maths/area.nim)
+
+## Sorts
+  * [Bubble sort](sorts/bubble_sort.nim)
```

Comment:
I just noticed that capitalization of titles is not coherent.

Reply:
That is something related to the directory script.
Another issue should be created for that, IMO.